test/widgets/emoji_reaction_test.dart

```diff
@@ -39,30 +42,24 @@ void main() {
 
     Future<void> setupChipsInBox(WidgetTester tester, {
       required List<Reaction> reactions,
-      double? width,
-      TextDirection? textDirection,
+      double width = 245.0, // (seen in context on an iPhone 13 Pro)
     }) async {
       final message = eg.streamMessage(reactions: reactions);
 
-      await tester.pumpWidget(
-        MaterialApp(
-          home: Directionality(
-            textDirection: textDirection ?? TextDirection.ltr,
```

Comment:
nit: currently these get turned into the `tester.platformDispatcher` logic in the big conversion commit:
c026fce27 test: Use TestZulipApp everywhere else it makes sense

Probably that'd be better as a separate commit coming after that one, since it's different from the otherwise-repetitive other changes in the commit.